pkg/xds/topology/outbound_test.go

```diff
@@ -1434,74 +1434,118 @@ var _ = Describe("TrafficRoute", func() {
 						},
 					},
 				},
-				mesh: defaultMeshWithMTLS,
+				zoneEgresses: []*core_mesh.ZoneEgressResource{
+					{
+						Meta: &test_model.ResourceMeta{
+							Name: "egress",
+							Mesh: "default",
+						},
+						Spec: &mesh_proto.ZoneEgress{
+							Networking: &mesh_proto.ZoneEgress_Networking{
+								Address: "1.1.1.1",
+								Port:    10002,
+							},
+						},
+					},
+				},
+				mesh: defaultMeshWithMTLSAndZoneEgress,
 				expected: core_xds.EndpointMap{
 					"another-mes": []core_xds.Endpoint{
 						{
-							Target: "example.com",
-							Port:   443,
+							Target: "1.1.1.1",
+							Port:   10002,
 							Tags: map[string]string{
 								"custom-label": "label",
 							},
 							Locality: nil,
 							Weight:   1,
 							ExternalService: &core_xds.ExternalService{
-								Protocol:                 core_mesh.ProtocolTCP,
-								TLSEnabled:               true,
-								FallbackToSystemCa:       true,
-								AllowRenegotiation:       false,
-								SkipHostnameVerification: true,
-								ServerName:               "example.com",
-								SANs:                     []core_xds.SAN{},
+								Protocol:   core_mesh.ProtocolTCP,
+								TLSEnabled: false,
 							},
 						},
 					},
 					"no-tls-mes": []core_xds.Endpoint{
 						{
-							UnixDomainPath: "unix://no-tls-mes",
```

Comment:
if it always goes through egress does unix domain socket make sense still? Should we just remove the support for it altogether?

Reply:
Maybe it doesn't make sense, It would require egress to run on the same machine. We can deprecate it in a separate pr 👍

Reply:
I'll open up an issue about this.